assignments/arrays.js

```diff
@@ -63,34 +314,45 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(inventory[32].car_make);
 
```

Comment:
Is there a more dynamic way to get the answer? What if the car with id: 25 is deleted from the array? Try to get this answer in a way that will find id 33 no matter what index it's stored at.